### PR TITLE
52  - Page title block

### PR DESCRIPTION
**Pull request by jbloomfield**

### CODE REVIEW

File: config/install/core.entity_view_display.node.localgov_publication_page.default.yml [hunk 1]
```diff
@@ -24,10 +24,18 @@ content:
     third_party_settings: {  }
     weight: 0
     region: content
+  field_localgov_services_landing:
```
Comment (rupertj) [inline config/install/core.entity_view_display.node.localgov_publication_page.default.yml:27]: This shouldn't have the field_ prefix. Just localgov. See here: https://github.com/localgovdrupal/localgov/wiki/Naming-conventions#set-field-prefix-to-localgov

File: localgov_publications.module [hunk 2]
```diff
@@ -61,6 +71,18 @@ function localgov_publications_theme_suggestions_book_navigation(array $variable
   return $suggestions;
 }
 
+/**
+ * Implements hook_block_access().
+ */
+function localgov_publications_block_access(Block $block, $operation, AccountInterface $account) {
+  if ($block->getPluginId() == 'localgov_page_header_block' && $operation == 'view') {
+    $node = \Drupal::routeMatch()->getParameter('node');
+    if (!empty($node) && $node instanceof NodeInterface && $node->getType() == 'localgov_publication_page') {
```
Comment (rupertj) [inline localgov_publications.module:80]: Is the check on empty() necessary? I'd have thought checking $node is an instance of NodeInterface here is enough.

**Review by rupertj — CHANGES_REQUESTED** [review]

File: config/install/core.entity_view_display.node.localgov_publication_page.default.yml (hunk 1 above)
Comment (finnlewis) [reply config/install/core.entity_view_display.node.localgov_publication_page.default.yml:27]: Ach - looks like this is the name of the field in https://github.com/localgovdrupal/localgov_services/blob/2.x/modules/localgov_services_landing/config/install/field.storage.node.field_localgov_services_landing.yml

Sorry, should have spotted that!

**Review by finnlewis — APPROVED** [review]
This is working as a page title block, but might raise some new questions about whether to use the title of the parent landing page if extant, and whether to display the title of the child pages on each child page. 

I think we merge this and test on an alpha release to see how it plays in real life scenarios.

**Comment by finnlewis** [comment]
And thanks @jbloomfield !

**Pull request merged**